### PR TITLE
Update Getting Started for Android

### PR DESCRIPTION
**Pull request by torn4dom4n**
Update **applicationId** and [build.gradle](https://github.com/appwrite/playground-for-flutter/blob/0fdbdff98384fff940ed0b1e08cf14cfe3a2be3e/android/app/build.gradle#L41) in `Getting Started for Android`. Follow [https://appwrite.io/docs/getting-started-for-flutter#android](https://appwrite.io/docs/getting-started-for-flutter)

### CODE REVIEW

File: app/views/docs/getting-started-for-android.phtml [hunk 1]
```diff
@@ -25,7 +25,7 @@ $androidVersion = (isset($versions['android'])) ? $versions['android'] : '';
 
 <p>To init your SDK and start interacting with Appwrite services, you need to add a new Android platform to your project. To add a new platform, go to your Appwrite console, choose the project you created in the step before, and click the 'Add Platform' button.</p>
 
-<p>From the options, choose to add a new <b>Android</b> platform and add add your app <u>name</u> and <u>package name</u>, Your package name is generally the applicationId in your app-level build.gradle file. By registering your new app platform, you are allowing your app to communicate with the Appwrite API.</p>
+<p>From the options, choose to add a new <b>Android</b> platform and add add your app <u>name</u> and <u>package name</u>, Your package name is generally the <b>applicationId</b> in your app-level <a href="https://github.com/appwrite/playground-for-flutter/blob/0fdbdff98384fff940ed0b1e08cf14cfe3a2be3e/android/app/build.gradle#L41" target="_blank" rel="noopener">build.gradle</a> file. By registering your new app platform, you are allowing your app to communicate with the Appwrite API.</p>
```
Comment (lohanidamodar) [inline app/views/docs/getting-started-for-android.phtml:28]: better to use link from playground-for-android

**Review by lohanidamodar — CHANGES_REQUESTED** [review]

File: app/views/docs/getting-started-for-android.phtml (hunk 1 above)
Comment (torn4dom4n) [reply app/views/docs/getting-started-for-android.phtml]: my bad

File: app/views/docs/getting-started-for-android.phtml [hunk 2]
```diff
@@ -25,7 +25,7 @@ $androidVersion = (isset($versions['android'])) ? $versions['android'] : '';
 
 <p>To init your SDK and start interacting with Appwrite services, you need to add a new Android platform to your project. To add a new platform, go to your Appwrite console, choose the project you created in the step before, and click the 'Add Platform' button.</p>
 
-<p>From the options, choose to add a new <b>Android</b> platform and add add your app <u>name</u> and <u>package name</u>, Your package name is generally the applicationId in your app-level build.gradle file. By registering your new app platform, you are allowing your app to communicate with the Appwrite API.</p>
+<p>From the options, choose to add a new <b>Android</b> platform and add add your app <u>name</u> and <u>package name</u>, Your package name is generally the <b>applicationId</b> in your app-level <a href="https://github.com/appwrite/playground-for-android/blob/5215c30a7242339606002e56dbe49d8a3e56a4fc/app/build.gradle#L11" target="_blank" rel="noopener">build.gradle</a> file. By registering your new app platform, you are allowing your app to communicate with the Appwrite API.</p>
```
[the remark this answers is not in the document]
Comment (lohanidamodar) [reply app/views/docs/getting-started-for-android.phtml:28]: It should rarely change, even if it changes, we can update the docs.

[the remark this answers is not in the document]
Comment (torn4dom4n) [reply app/views/docs/getting-started-for-android.phtml:28]: 👍

**Review by lohanidamodar — APPROVED** [review]

**Comment by christyjacob4** [comment]
THANK YOU! All changes merged 🥳

Please reach out to me on our [Discord server](https://appwrite.io/discord) if you would like to claim your Appwrite swags! As a way of saying thank you, we would also love to invite you to join the Appwrite organization on GitHub. Please share your GitHub username with us on Discord.  

You can accept the invite by visiting https://github.com/orgs/appwrite/invitation. By joining our team, you will officially be an Appwrite maintainer on GitHub.

You can change your membership [visibility settings](https://help.github.com/en/github/setting-up-and-managing-your-github-user-account/publicizing-or-hiding-organization-membership), so your new Appwrite team membership badge will show up on your personal GitHub profile.

Please feel free to look for more PRs you might be interested in helping with on our long list of [Hacktoberfest friendly issues](https://github.com/search?q=org%3Aappwrite+org%3Autopia-php+is%3Aissue+label%3Ahacktoberfest+created%3A%3E2021-01-01+is%3Aopen&type=issues) and help make Appwrite better :)

**Pull request merged**